aws/logs_monitoring/lambda_function.py

```diff
@@ -390,6 +425,29 @@ def generate_metadata(context):
     return metadata
 
 
+def forward_lambda_metrics(logs):
+    """
+    Process custom metrics submitted via lambda logs, and send them to Datadog
+    in a background thread using `lambda_metric` that is provided by the
+    Datadog Python Lambda Layer.
+    """
+    if not DD_FORWARD_METRIC:
+        return
+
+    for log in logs:
+        try:
+            if log.get(DD_SOURCE) != 'lambda':
+                continue
+            metric = json.loads(log['message'])
+            required_attrs = ('metric_name', 'value', 'timestamp', 'tags')
+            if all(attr in metric for attr in required_attrs):
+                lambda_metric(**metric)
```

Comment:
`lambda_metric` queues the metrics and flush them in another python thread periodically.

Reply:
can we use a more explicit name or use your message as comment please ? What about `lambda_metric.push(**metric)` or `lambda_metric.add(**metric)` or `forward_to_lambda_metric(**metric)`

Reply:
Yes agreed. I will add a comment. `lambda_metric` is provided by lambda layer which is used by customers too.